resources/lang/en/notifications.php

```diff
@@ -51,6 +51,17 @@
             ],
         ],
 
+        'channel' => [
+            '_' => 'Chat',
+
+            'channel' => [
+                '_' => 'New message',
+                'channel_message' => ':username says ":title"',
+                'channel_message_compact' => ':title',
+                'channel_message_group' => 'from :username',
```

Comment:
this looks like specific for pm instead of channel message? (also in some other places like url generator)